distributed/tests/test_steal.py

```diff
@@ -1361,6 +1356,7 @@ async def test_reschedule_concurrent_requests_deadlock(c, s, *workers):
     assert msgs in (expect1, expect2, expect3)
 
 
+@pytest.mark.skip("executing heartbeats not considered yet")
```

Comment:
This also seems important to fix before a release. I have a feeling that main use cases where people actually depend on stealing right now is submitting a bunch of very, very slow tasks, then scaling up the cluster.

Reply:
See https://github.com/dask/distributed/pull/7030#discussion_r971979808 for @fjetter's thoughts on this.